configs/chess.yaml

```diff
@@ -25,7 +25,7 @@ game:
             dataset_args:
               filename: ./data/chess/stockfish_examples.jsonl
               role: White
-              num_examples: 10
+              num_samples: 10
```

Comment:
does this change need to be made in all configs?

Reply:
This is only for this particular dataset class. We could have this as an argument for the super class though and have that functionality built-in to all datasets. Wdyt?

Reply:
yeah, let's do that in the next iteration. I just reverted all the name changes and tested.